modules/storages/app/common/storages/peripherals/nextcloud_connection_validator.rb

```diff
@@ -54,10 +54,26 @@ def validate
 
       private
 
+      def has_oidc_configuration_error?
```

Comment:
:yellow_circle: Since the method name ends in a `?`, I'd expect that I can interpret the result of this method as a boolean. However, as far as I can tell, both `None` and `Some` are truthy values.

Since this is used in the configuration checks below, what do you think about

```suggestion
      def sso_is_misconfigured
```

1. Rephrased to be part of a sentence ("or SSO is misconfigured")
2. Replacing OIDC with SSO to try and achieve consistency in naming with regards to that (Nit)

Reply:
About the the query method, I was just following the pattern already existing in the object itself.

Reply:
I see, because there are two methods that also end in `?` while returning some or none. IMO we should remove the question mark from those two methods as well, all the others already behave nicely.

Reply:
Ended up going with your suggestion. Let's see how this shapes up as I implement the rest of the checks. =)